packages/preview/hsrm-bachelor-thesis/0.6.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Lasse Rosenow
```

Comment:
You probably want to change the year and name here. If you based your templates on someone else's, I think you are supposed to keep their copyright info too, but I'm not 100% sure, don't quote me on that.